installer/charts/rhtap-dh/templates/extra-env.yaml

```diff
@@ -59,7 +59,14 @@ data:
 {{- end }}
 {{- $quaySecretObj := (lookup "v1" "Secret" .Release.Namespace "rhtap-quay-integration") }}
 {{- $quaySecretData := ($quaySecretObj.data | default dict) }}
-{{- if $quaySecretData.token }}
+{{- if $quaySecretData }}
+    {{- if $quaySecretData.token }}
     QUAY__API_TOKEN: {{ $quaySecretData.token }}
-{{- end }}
+    {{- end }}
     QUAY__URL: {{ $quaySecretData.url }}
+{{- end }}
+{{- $nexusSecretObj := (lookup "v1" "Secret" .Release.Namespace "rhtap-nexus-integration") }}
+{{- $nexusSecretData := ($nexusSecretObj.data | default dict) }}
+{{- if $nexusSecretData }}
+    NEXUS__URL: {{ $nexusSecretData.url }}
```

Comment:
I'm surprised there is no token for private image repositories. OK to merge as is until we figure it out.